### ISSUE TITLE
Add cross-validation curve for appointed parameter of param_range

### ISSUE THREAD

**Issue by lugq1990** [reiinakano/scikit-plot#80]
@reiinakano Hi,  I have just found that the project of scikit-plot is really helpful for those what to do the data analysis or machine learning, and I use it much. During my working time, I find that for the parameter choosen maybe also be plot for visualization. I have writen a new method for plot the cross-validation for a appointed parameter. I want to create a new branch added the new added method. Is that ok?

**Comment by reiinakano** [comment]
Hi @lugq1990. Contributions are welcome. Do you mind posting here your proposed plot and some examples?

**Comment by lugq1990** [comment]
I have writen the function ready, such as this. The final result is the plot image. I am wondered if you would see it. As the link.
`from __future__ import absolute_import
from sklearn.datasets import load_digits as load_data
from sklearn.utils import shuffle
import matplotlib.pyplot as plt
import scikitplot as skplt
from sklearn.svm import SVC
X, y = load_data(return_X_y=True)
X, y = shuffle(X, y)
clf = SVC()
param_name = 'C'
param_range = [.1,10,100]
skplt.estimators.plot_validation_curve(clf, X, y, param_name=param_name, param_range=param_range)
plt.show()`
![image](https://user-images.githubusercontent.com/29392424/35956719-6aaf1abc-0cd2-11e8-8632-7f52c542bac4.png)
